dev/io.openliberty.threading.internal.java21/.project

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This should be removed.  Eclipse added it back to your environment and you added it to your PR erroneously.